lib/ansible/module_utils/hwc_utils.py

```diff
@@ -68,6 +68,21 @@ def replace_resource_dict(item, value):
         return item.get(value)
 
 
+def is_different(expect, actual):
```

Comment:
Now that this is in module_utils, it probably makes sense to rename it to something less generic, like `are_dicts_different`, so that users know this is about `dict`s and not about arbitrary objects.

Reply:
Done